Elsewhen/Elements/DateTimeSelection.swift

```diff
@@ -206,7 +204,7 @@ struct DateTimeSelection: View, OrientationObserving {
                 
             } else {
                 
-                DateTimeZonePicker(selectedDate: $selectedDate, selectedTimeZone: $selectedTimeZone, maxWidth: dateFormatsMaxWidth)
+                DateTimeZonePicker(selectedDate: $selectedDate, selectedTimeZone: $selectedTimeZone, maxWidth: 376)
```

Comment:
I haven't run this to see how it looks, but why replace a dynamic value with a constant? Also, if they are intended to be the same then why not assign them both to a static variable so they definitely stay the same?
How does it look on mac? This is the sort of thing that tends to translate poorly.

Reply:
Because the dynamic sizing was "make the Time Zone choice view as wide as the buttons beneath it" so on iPad they stretched to fill the entire width - but the Date and Time picker above it didn't, so it looked weird. Giving them all the same max width means they match the width of the Date and Time picker above and it looks much neater.

e.g. before:

<img width="722" alt="Screenshot 2022-06-09 at 19 59 30" src="https://user-images.githubusercontent.com/882469/172923776-6a8cda41-2a88-47af-a6c9-3450e622f765.png">

after:

<img width="763" alt="Screenshot 2022-06-09 at 20 00 16" src="https://user-images.githubusercontent.com/882469/172923908-3ca135d7-857d-4063-b0f9-37da94933c0f.png">

macOS stays the same, because that doesn't use this code path:

<img width="900" alt="Screenshot 2022-06-09 at 20 03 35" src="https://user-images.githubusercontent.com/882469/172924463-c05173ba-a096-4d0a-a759-db2685b77535.png">